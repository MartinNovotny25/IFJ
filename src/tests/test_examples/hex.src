a = '\A'
print(a)
a = '\x21' #!
b = '\x2a' #*
c = '\x2A' #*
e = '\x7a' #z
f = '\x7E' #~
d = '\X2A' #\X2A
print(d)
g = '\Aaa'
h = '\AAA'
i = '\ZFF'
i = '\zFF'
print(f)
