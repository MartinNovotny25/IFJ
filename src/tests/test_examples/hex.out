TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \A)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, !)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, *)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, *)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, z)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, ~)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \X2A)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \Aaa)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \AAA)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \ZFF)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \zFF)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOF)
